.eslintrc.js

```diff
@@ -45,8 +58,12 @@ module.exports = {
     '@intlify/vue-i18n/no-raw-text': 'off',
     'vue/multi-word-component-names': 'off',
     'vue/no-v-model-argument': 'off',
+    'vue/no-reserved-component-names': 'off',
     '@typescript-eslint/no-unused-vars': ['error', { ignoreRestSiblings: true }],
     '@typescript-eslint/member-delimiter-style': ['error'],
+    'default-param-last': 'off',
```

Comment:
I don't see the reason for disabling this rules as putting the default params as the last one is really useful.

Reply:
We have the cases where we have a default parameter second, and optional third.